doc/src/installation.rst

```diff
@@ -10,85 +10,55 @@ including Apple OS X, but they are not officially tested and supported.
 Third-Party Software Packages
 -----------------------------
 
-**Python 2** ``>=`` **2.6** is required. **Python 2** ``>=`` **2.7.9** is
-recommended for the best security. `Python <https://python.org/>`_ 2 should
-already be installed in your Linux system.
+Requirements:
 
-For Cylc's HTTPS communications layer:
+- Python 3.6+
```

Comment:
Is this what we are going for?

Reply:
It's not what we're going for but I think it's what currently works!